src/platform_impl/ios/event_loop.rs

```diff
@@ -94,6 +95,11 @@ impl<T: 'static> EventLoopWindowTarget<T> {
     debug!("`EventLoopWindowTarget::cursor_position` is ignored on iOS");
     Ok((0, 0).into())
   }
+
+  /// Sets badge count on iOS launcher. 0 hides the count
+  pub fn set_badge_count(&self, count: i32) {
```

Comment:
this needs to be exposed on `EventLoopWindowTargetExtIOS` as well as `WindowExtIOS`

Reply:
I shall do it

Reply:
There's no `EventLoopWindowTargetExtIOS`